### PR TITLE
feat(common): add additional tests for default storage class

### PR DESCRIPTION
**Pull request by Ornias1993**
**Description**

⚒️ Fixes  # 

**⚙️ Type of change**

- [ ] ⚙️ Feature/App addition
- [ ] 🪛 Bugfix
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🔃 Refactor of current code

**🧪 How Has This Been Tested?**

**📃 Notes:**

**✔️ Checklist:**

- [ ] ⚖️ My code follows the style guidelines of this project
- [ ] 👀 I have performed a self-review of my own code
- [ ] #️⃣ I have commented my code, particularly in hard-to-understand areas
- [ ] 📄 I have made corresponding changes to the documentation
- [ ] ⚠️ My changes generate no new warnings
- [ ] 🧪 I have added tests to this description that prove my fix is effective or that my feature works
- [ ] ⬆️ I increased versions for any altered app according to semantic versioning
- [ ] I made sure the title starts with `feat(chart-name):`, `fix(chart-name):` or `chore(chart-name):`

**➕ App addition**

If this PR is an app addition please make sure you have done the following.

- [ ] 🖼️ I have added an icon in the Chart's root directory called `icon.png`

---

_Please don't blindly check all the boxes. Read them and only check those that apply.
Those checkboxes are there for the reviewer to see what is this all about and
the status of this PR with a quick glance._

### CODE REVIEW

File: library/common-test/tests/storageClass/metadata_test.yaml [hunk 1]
```diff
@@ -70,3 +70,32 @@ tests:
             label1: label1
             label2: label2
             some-label: some-value
+  - it: should pass with storageClass default set to true
```
Comment (stavros-k) [inline library/common-test/tests/storageClass/metadata_test.yaml:73]: A test for that should already exist in the spec_test file!

Comment (Ornias1993) [reply library/common-test/tests/storageClass/metadata_test.yaml:73]: what is it doing in spec at all?
Its a metadata/annotation thing after all.

Comment (stavros-k) [reply library/common-test/tests/storageClass/metadata_test.yaml:73]: Probably by mistake. you can remove it from there

Comment (Ornias1993) [reply library/common-test/tests/storageClass/metadata_test.yaml:73]: will do :)

**Review by stavros-k — APPROVED** [review]

**Pull request merged**